funnel/views/proposal.py

```diff
@@ -62,9 +62,9 @@ def proposal_data(proposal):
             ('objective', proposal.objective.html),
             ('description', proposal.description.html),
             ('requirements', proposal.requirements.html),
-            ('slides', proposal.slides.url),
+            ('slides', proposal.slides.url if proposal.slides is not None else ''),
```

Comment:
This can be achieved with just `or ''`

Reply:
That will break if `proposal.slides` is missing/`None`. which is why I added it.

Reply:
That is what `or ''` solves for.

Reply:
`proposal.slides.url` raises `AttributeError` is `proposal.slides` is not set.

Reply:
Ah, missed the `.url` bit.